aws/resource_aws_appmesh_virtual_node.go

```diff
@@ -44,9 +51,36 @@ func resourceAwsAppmeshVirtualNode() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"backends": {
 							Type:     schema.TypeSet,
+							Removed:  "Use `backend` configuration blocks instead",
```

Comment:
Not going through `Deprecated` cycle here, straight to `Removed` as AppMesh is a preview service.